WordPressKit/WordPressKit/ActivityServiceRemote.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+import WordPressShared
+import CocoaLumberjack
+
+public class ActivityServiceRemote: ServiceRemoteWordPressComREST {
+
+    public enum ResponseError: Error {
+        case decodingFailure
+    }
+
+    /// Retrieves activity events associated to a site.
+    ///
+    /// - Parameters:
+    ///     - siteID: The target site's ID.
+    ///     - offset: The first N activities to be skipped in the returned array.
+    ///     - count: Number of objects to retrieve.
```

Comment:
Just leaving a note here that I don't think offset/count pagination will be a good fit for this, but that discussion belongs elsewhere 😁